system/jlib/jptree.cpp

```diff
@@ -2572,6 +2572,10 @@ void PTree::addLocal(size32_t l, const void *data, bool _binary, int pos)
     IPTArrayValue *newValue = new CPTValue(l, data, _binary);
     Owned<IPropertyTree> tree = create(queryName(), newValue);
     PTree *_tree = QUERYINTERFACE(tree.get(), PTree); assertex(_tree); _tree->setParent(this);
+    if (_binary)
+        _tree->flags |= ipt_binary;
+    else
+        _tree->flags &= ~ipt_binary;
```

Comment:
Why not, just for consistency, use IptFlagSet(_tree->flags, ipt_binary) here ?

Reply:
You're right, I should. I will change.

Reply:
changed